notes/anderson.md

```diff
@@ -0,0 +1,3 @@
+# Anderson's Update
+
+[EPF Notes 20221026](https://hackmd.io/@duE1tlDcSzSfvg9NCpsgfw/BkCypv5Es)
```

Comment:
Thanks! 

Can you also add this update in the [Development Updates](https://github.com/eth-protocol-fellows/cohort-three/blob/master/development-updates.md) document?